03-intro-to-python.Rmd

```diff
@@ -1,9 +1,298 @@
 
 # Intro to Python
 
+Python is a popular programming language that was created by Guido van Rossum and released in 1991. 
+
+Python is supported by multiple libraries that support data science tasks:
+
+- NumPy for numerical computing with multidimensional arrays
+- pandas for data manipulation and analysis with data frames
+- Matplotlib for data visualization
+
+## Main Differences between R and Python
+
+The main difference between Python and R is that Python is a general-purpose programming language, while R is a statistical programming language. This means that Python is good at multiple things and can do most things, whereas R is very good at statistical analysis, but not as good at other things as Python.
+
+When you want to document your code, you can use Jupyter Notebooks. Jupyter Notebooks are an open source web application for easily sharing documents that contain your live Python code, equations, visualizations and data science explanations. 
+
+Python is particularly suited for large scale machine learning and deep learning with libraries such as TensorFlow, PyTorch, and scikit-learn.
+
 
 ## Learning Objectives
 
 ```{r, fig.align='center', echo = FALSE, fig.alt= "Major point!! example image"}
```

Comment:
I found that using `![](https://docs.google.com/presentation/d/1k8uC1rqnGTSbKjBsWvKYgiUUxO1q_VhJCwZQHJNWozA/export/png?pageid=g29054a882fd_0_52)` also does the trick. Could be worth considering because we are then only using standard markdown syntax without relying on having an R interpreter available (e.g., if we want to open these documents on Colab as discussed in #6). The syntax is

```markdown
![](https://docs.google.com/presentation/d/[presentation id here]/export/png?pageid=[page id here without the 'id.' prefix]
```

<details><summary>Example</summary>

![](https://docs.google.com/presentation/d/1k8uC1rqnGTSbKjBsWvKYgiUUxO1q_VhJCwZQHJNWozA/export/png?pageid=g29054a882fd_0_52)
</details>